nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestDeduplicateRecord.java

```diff
@@ -271,7 +271,7 @@ void doCountTests(int failure, int original, int duplicates, int notDuplicates,
         }
     }
 
-    private static final class MockCacheService<K, V> extends AbstractControllerService implements DistributedMapCacheClient {
+    static final class MockCacheService<K, V> extends AbstractControllerService implements DistributedMapCacheClient {
```

Comment:
Is this change necessary as part of this PR?

Reply:
I did use it in my tests in order to test the listing "AbstractListProcessor.BY_ENTITIES" option. I could try moving the MockCacheService to a separate class that both test classes could use?

Reply:
Yes, moving it to a shared class seems like the best approach.